src/Microsoft.VisualStudio.Web.CodeGeneration.Design/Program.cs

```diff
@@ -43,13 +43,22 @@ private static void Execute(string[] args, ConsoleLogger logger)
             var noBuild = app.Option("--no-build", "", CommandOptionType.NoValue);
             var simMode = app.Option("--simulation-mode", Resources.SimulationModeOptionDesc, CommandOptionType.NoValue);
 
-#if DEBUG
-            if (args.Contains("--debug", StringComparer.OrdinalIgnoreCase))
+// the for reals version
+//
+//#if DEBUG
+//            if (args.Contains("--debug", StringComparer.OrdinalIgnoreCase))
```

Comment:
Looks like we need to evolve this condition to support debugging when testing using VS. 
Maybe along with the parameter, look for an environment variable?